gensim/models/ldamodel.py

```diff
@@ -567,15 +567,18 @@ def init_dir_prior(self, prior, name):
         if isinstance(prior, six.string_types):
             if prior == 'symmetric':
                 logger.info("using symmetric %s at %s", name, 1.0 / self.num_topics)
-                init_prior = np.asarray([1.0 / self.num_topics for i in xrange(prior_shape)], dtype=self.dtype)
+                init_prior = np.fromiter((1.0 / self.num_topics for i in xrange(prior_shape)),
+                                            dtype=self.dtype, count=prior_shape)
```

Comment:
Hanging indent please, here and elsewhere.

We avoid vertical indent because it leads to inconsistent visual indenting (like here), especially when the lines are edited in the future.

Reply:
Done.